scripts/split-paired-reads.py

```diff
@@ -128,47 +131,48 @@ def main():
         # Use default filename created above
         fp_out2 = get_file_writer(open(out2, 'wb'), args.gzip, args.bzip)
 
+    # put orphaned reads here, if -0!
+    allow_orphans = False
```

Comment:
On Tue, Aug 04, 2015 at 09:52:19AM -0700, Michael R. Crusoe wrote:
> > @@ -128,47 +131,48 @@ def main():
> >          # Use default filename created above
> >          fp_out2 = get_file_writer(open(out2, 'wb'), args.gzip, args.bzip)
> >  
> > +    # put orphaned reads here, if -0!
> > +    allow_orphans = False
> 
> Why not use `args.output_orphaned` everywhere you check `allow_orphans`?

It's partly a leftover of some now-removed logic that permitted -0 without
a filename, but what made me keep it in was this code:

`require_paired=not allow_orphans`

which I think would be uglier and less clear if it was
`not args.output_orphaned`.  What do you think?